css/style.css

```diff
@@ -34,7 +35,7 @@ header {
   font-size: 1.5rem;
 }
 
-main {
+.mx-container {
```

Comment:
- [ ] [OPTIONAL] The 'About us' section should have a `white` backgroud with rounded top-right corner according to [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Z6DrF2ZhjU2Iosol-0). I highly recommend adding a `white` background with rounded corner instead of `grey` background. As the [requirement of the project](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements) is to make an exact replica of the[ figam design template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=Z6DrF2ZhjU2Iosol-0).

**_Figma_** 

<img width="146" alt="Screenshot 2023-02-25 at 7 56 07 AM" src="https://user-images.githubusercontent.com/44798044/221332840-3de3d7ad-5ea1-454d-a34a-4413ac8ef683.png">

**_Yours_**

<img width="304" alt="Screenshot 2023-02-25 at 7 56 28 AM" src="https://user-images.githubusercontent.com/44798044/221332852-7c6e277b-41b5-432b-b28c-5d09bd6a6c90.png">